airbyte-integrations/connectors/source-snapchat-marketing/source_snapchat_marketing/spec.json

```diff
@@ -45,6 +45,14 @@
         "examples": ["2022-01-30"],
         "order": 4,
         "format": "date"
+      },
+      "report_dimension":  {
```

Comment:
field that present an list of dimensions should be array with enum to avoid typo in dimensions name

Reply:
@lazebnyi Apologies for the delay, the PR is ready for review.